swagger-api.yaml

```diff
@@ -1,233 +1,233 @@
-      tags:
-        - account
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/AccountUpdate'
-        required: true
-      responses:
-        200:
-          description: 'Successfully updated'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/AccountInfo'
-        400:
-          description: 'Bad request'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        401:
-          description: 'Authenticate failed'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        404:
-          description: 'Account does not exist'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        default:
-          description: 'Unexpected error'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-      security:
-        - basicAuth: []
-    delete:
-      summary: 'Delete account'
-      description: 'Delete account.'
-      tags:
-        - account
-      responses:
-        204:
-          description: 'Successfully deleted'
-        401:
-          description: 'Authenticate failed'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        404:
-          description: 'Account does not exist'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        default:
-          description: 'Unexpected error'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-      security:
-        - basicAuth: []
-
-components:
-  securitySchemes:
-    basicAuth:
-      type: http
-      scheme: basic
-  schemas:
-    AccountCreate:
-      required:
-        - username
-        - password
-        - email
-      properties:
-        username:
-          type: string
-        password:
-          type: string
-        email:
-          type: string
-    AccountUpdate:
-      properties:
-        password:
-          type: string
-        email:
-          type: string
-        firstName:
-          type: string
-        lastName:
-          type: string
-        phone:
-          type: string
-    AccountInfo:
-      properties:
-        id:
-          type: string
-        username:
-          type: string
-        email:
-          type: string
-        firstName:
-          type: string
-        lastName:
-          type: string
-        phone:
-          type: string
-    Error:
-      properties:
-        code:
-          type: string
-        message:
-          type: string
+openapi: "3.0.0"

```

Comment:
What are changes in this file?

Reply:
Nothing. It is connected with type of line ending ("lf" or "crlf"). I fixed it with file .gitattributes

Reply:
So what's changed in the file, why git shows diffs?

Reply:
Rollbacked changes in swagger-api.yaml (my IDE changed line endings CRLF).
Fixed.
Thanks

Reply:
I'm still wondering, what was changed? 🤔

Reply:
To my mind this file was changed because I used Goland in Windows OS. I did not change swagger-api.yaml.